### PR TITLE
Added cola-compound demo #1043

### PR DESCRIPTION
**Pull request by mj3cheun**

### CODE REVIEW

File: documentation/demos/cola-compound/cytoscape-cola.js [hunk 1]
```diff
@@ -0,0 +1,549 @@
+/*!
```
Comment (maxkfranz) [inline documentation/demos/cola-compound/cytoscape-cola.js]: This file should not be included, nor the Cola lib itself

File: documentation/demos/cola-compound/index.html [hunk 2]
```diff
@@ -0,0 +1,1223 @@
...
+						css: {
+							'background-color': '#aaa',
+							'line-color': 'black',
+							'target-arrow-color': 'black',
+							'source-arrow-color': 'black'
+						}
+					}
+				],
+
+				elements: {
```
Comment (maxkfranz) [inline documentation/demos/cola-compound/index.html]: Please put the graph in a separate json file to keep the demo easier to read.

File: documentation/demos/cola-compound/index.html [hunk 3]
```diff
@@ -0,0 +1,1223 @@
...
+				container: document.getElementById('cy'),
+
+				layout: {
+					name: 'cola'
+				},
+
+				style: [{
+						selector: 'node',
+						css: {
+							'content': 'data(label)',
```
Comment (maxkfranz) [inline documentation/demos/cola-compound/index.html]: Please use 2 spaces rather than tabs for indentation.

**Comment by maxkfranz** [comment]
Great.  Could we use a simpler dataset like the one here? http://marvl.infotech.monash.edu/webcola/examples/gridifiedSmallGroups.html

**Comment by mj3cheun** [comment]
I couldn't find any compound graph smaller in in Dylan's graph list or in Pathway Commons so I just started deleting nodes from the existing graph. The graph probably doesn't make sense biologically anymore but its smaller. Also added ES6 polyfills just in case.

**Comment by maxkfranz** [comment]
Could we use the same data from http://marvl.infotech.monash.edu/webcola/examples/gridifiedSmallGroups.html ? It would be good for it to be comparable to the Cola example

**Comment by mj3cheun** [comment]
As far as I can tell, they use a different data format. I guess I could figure out how to reconstruct that dataset though.

**Comment by maxkfranz** [comment]
OK, great.  Worst case, it's not many nodes

**Pull request merged**